Gemfile

```diff
@@ -1,3 +1,8 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :acceptance do
+  gem 'puppetlabs_spec_helper'
+  gem 'puppet_litmus'
+end
```

Comment:
you can remove this

Reply:
Don't know how I missed to take that out... @chelnak thanks for keeping me right